lib/libyui/tasks.rb

```diff
@@ -65,6 +65,26 @@ def cmake_version(file = nil)
          cmake_value(f, "VERSION_PATCH"),
          cmake_value(f, "VERSION_TWEAK")].compact.join(".")
       end
+
+      # Returns the so version
+      #
+      # If no filename is given, it searches for the so version in all the spec files
+      #
+      # @param filename [String, nil]
+      # @return [String, nil] nil if no so version was found
+      def so_version(filename = nil)
+        filenames = Dir.glob("package/*.spec").sort
+        filenames = [ filename ] if filename
+
+        version = nil
+
+        filenames.each do |filename|
+          version = File.read(filename).scan(/^%define.*so_version.([\d]+)/).flatten.first
```

Comment:
Small fix for the regexp: 
```suggestion
          version = File.read(filename).scan(/^%define\s+so_version.(\d+)/).flatten.first
```